code/modules/recycling/sortingmachinery.dm

```diff
@@ -393,3 +393,6 @@
 	if(trunk)
 		trunk.linked = null
 	return ..()
+
+/obj/machinery/disposal/deliveryChute/full // Full variant that blocks air.
```

Comment:
I really do not like this. It's not intuitive that it blocks air, cause it has same sprite as a normal disposal chute that does not block air. Things should not have different behavior without changing their looks (name, description).

And also like, I guarantee people would use this in places that do not need the air-blocking disposals variant.

It have a new sprite that takes up a full tile and unambigiously looks like it blocks air.

Reply:
![image](https://github.com/Aurorastation/Aurora.3/assets/107256943/487620cc-ecff-40f1-a980-d451504feb4e)

Alternatively, if you can't get a new sprite, there could just be a glass pane between the disposals chute and the "door", so that air does not leak between the "airlock" and the chute.

Reply:
![image](https://github.com/Aurorastation/Aurora.3/assets/107256943/561c179a-06a5-4d9f-bd0e-9ba2c6f71735)

Now I looked up how it was before... Yeah.